FirebaseInstallations/Source/Tests/Utils/FIRKeydArchivingUtils.m

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "FIRKeydArchivingUtils.h"
+
+@implementation FIRKeydArchivingUtils
+
++ (nullable NSData *)archivedDataWithRootObject:(id)object error:(NSError **)outError {
+  NSData *archivedData;
+  if (@available(macOS 10.13, iOS 11.0, tvOS 11.0, *)) {
```

Comment:
Should we just start with higher os support considering this is a new SDK?

Reply:
I'll be happy to get rid of such conditions if it is an option.
@ryanwilson @paulb777 WDYT?

Reply:
although we might need to cover the minimum iOS version for Firebase overall, which is iOS 8? In this case, maybe we still need to keep this logic?

Reply:
so far, yes, this is the main motivation. FCM and Analytics (the main clients of FIS) support iOS 8 and higher

Reply:
Let's keep it then.